content/docs/gke/deploy/minikf-gcp.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploy using MiniKF on GCP"
+description = "Instructions for using the Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)."
+weight = 4
++++
+
+This guide describes how to launch a MiniKF VM on GCP. MiniKF is a
+production-ready, full-fledged Kubeflow deployment that installs in minutes.
+It's a fast and easy way to get started with Kubeflow. With just a few clicks,
+you are up for experimentation, and for running complete Kubeflow Pipelines.
+
+MiniKF installs Kubeflow 0.7.0 and supports NVIDIA GPUs.
+
+### Installing MiniKF on GCP
+
+To install MiniKF on GCP, follow the steps below:
+
+Go to the [MiniKF page](To install MiniKF on GCP, follow the steps below) on
+Google Cloud Marketplace.
+
+Click on the Launch on Compute Engine button
+
+<p class="text-center">
+  <img src="/docs/images/minikf-launch.png" alt="Launch MiniKF on GCP">
+</p>
+<p class="text-center" style="font-size:0.75rem">Figure 1: Launch MiniKF on GCP</p>
+
+In the Configure & Deploy window, choose:
+
+  - **name**: a name for your MiniKF instance
+  - **zone**: the GCP zone that you will deploy MiniKF
+  - **machine type**: a machine type with at least 2 vCPUs and 16 GB of RAM. We
+    propose using the default machine type, n1–standard-8, which has 8 vCPUs and
+    30 GB of RAM.
+  - **boot disk**: A boot disk. This is the disk that holds the images that MiniKF
+    needs. We propose using the default value of 200 GB SSD Persistent Disk.
```

Comment:
Replace "We propose using" with "Use" or with "If in doubt, use".

(It's tech writing best practice to avoid "we" and to just tell the user directly what to do.)

Reply:
Good to know, thank you